### PR TITLE
Split Upload Transfer Procedure into creation and appending procedures

### PR DESCRIPTION
**Pull request by guoye-zhang**

### CODE REVIEW

File: draft-tus-httpbis-resumable-uploads-protocol.md [hunk 1]
```diff
@@ -312,14 +303,53 @@ If the server does not consider the upload associated with this token active, it
...
 ~~~
 
-The client MAY automatically start uploading from the beginning using Upload Transfer Procedure ({{upload-transfer}}) if `404 (Not Found)` status code is received. The client SHOULD NOT automatically retry if a status code other than 204 and 404 is received.
+The client MAY automatically start uploading from the beginning using Upload Creation Procedure ({{upload-creation}}) if `404 (Not Found)` status code is received. The client SHOULD NOT automatically retry if a status code other than 204 and 404 is received.
+
+# Upload Appending Procedure {#upload-appending}
+
+The Upload Appending Procedure is used for resuming an existing upload.
+
+The request MUST use the `HEAD` method and include the `Upload-Token` header. The `Upload-Offset` header field ({{upload-offset}}) MUST be set to the resumption offset.
```
Comment (smatsson) [inline draft-tus-httpbis-resumable-uploads-protocol.md]: Typo? Should this be PATCH method?

File: draft-tus-httpbis-resumable-uploads-protocol.md [hunk 2]
```diff
@@ -312,14 +303,53 @@ If the server does not consider the upload associated with this token active, it
...
+
+If the end of the request body is not the end of the upload, the `Upload-Incomplete` header field ({{upload-incomplete}}) MUST be set to true.
+
+The client SHOULD NOT send representation metadata in the request and servers SHOULD ignore any representation metadata received.
```
Comment (smatsson) [inline draft-tus-httpbis-resumable-uploads-protocol.md]: I'm not saying it's wrong but isn't this kind of strange when considering that we mention "Content-Type" as metadata earlier in the draft? Content-Type is most likely automatically added when appending data by the underlying http lib.

File: draft-tus-httpbis-resumable-uploads-protocol.md (hunk 1 above)
Comment (guoye-zhang) [reply draft-tus-httpbis-resumable-uploads-protocol.md]: Fixed

File: draft-tus-httpbis-resumable-uploads-protocol.md (hunk 2 above)
Comment (guoye-zhang) [reply draft-tus-httpbis-resumable-uploads-protocol.md]: I removed the client-side requirement

**Review by smatsson — APPROVED** [review]

**Pull request merged**